guardian/gu-who
# Support alternative default branch names other than master

## What does this change?

It removes the hardcoding of the people branch to master. The Guardian is moving the default branch name used in it's repos from master to main. See [here](https://github.com/guardian/master-to-main) for more detail. 

## How to test

I have written a test against a local repo to verify the change works against a main default repo. 

## How can we measure success?

If the system continues to work for both master and main repos then it will have been successful. 

## Have we considered potential risks?

We cannot fully test it against PROD until we change the people [repo](https://github.com/guardian/people) to default to main. 
We need to be careful of any repos that don't have master set as the default branch.
If the code doesn't work as expected it could break the gu-who system in general. 


## Images
n/a
